STYLE_GUIDE.md

```diff
@@ -0,0 +1,25 @@
+# DELV Solidity Style Guide
+
+## Commenting
+
+1. Use complete sentences and end them with the appropriate puntuaction (typically a period).
+2. Use correct spelling.
+3. Use correct capitalization.
+4. Comments should not go past the 80th column from the start of the line.
+5. Add  full Natspec to each contract, library, function, storage variable, struct, event, and error.
```

Comment:
```suggestion
5. Add full Natspec to each contract, library, function, storage variable, struct, event, and error.
```

Reply:
One way to make this a bit more clear would be to highlight which [Natspec tags](https://docs.soliditylang.org/en/latest/natspec-format.html#tags) we would like to see for each context.

Reply:
oh good call. for example @notice on public/external and @dev for private/internal

Reply:
Oh good call. Yeah, I’ll add some more explicit rules for Natspec